fvtest/compilerunittest/p/BinaryEncoder.cpp

```diff
@@ -3878,6 +3890,15 @@ INSTANTIATE_TEST_CASE_P(LoadIndex, PPCTrg1MemEncodingTest, ::testing::ValuesIn(*
     std::make_tuple(TR::InstOpCode::lxvw4x,  TR::RealRegister::vsr0,  MemoryReference(TR::RealRegister::gr0,  TR::RealRegister::gr31), 0x7c00fe18u, false)
 )));
 
+INSTANTIATE_TEST_CASE_P(LoadIndex, PPCTrg1Src2EncodingTest, ::testing::Values(
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr0,  TR::RealRegister::gr0,  TR::RealRegister::gr0,  0x7c0006d8u),
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr31, TR::RealRegister::gr0,  TR::RealRegister::gr0,  0x7fe006d8u),
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr63, TR::RealRegister::gr0,  TR::RealRegister::gr0,  0x7fe006d9u),
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr63, TR::RealRegister::gr31, TR::RealRegister::gr31, 0x7ffffed9u),
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr0,  TR::RealRegister::gr31, TR::RealRegister::gr0,  0x7c1f06d8u),
+    std::make_tuple(TR::InstOpCode::lxvb16x, TR::RealRegister::vsr0,  TR::RealRegister::gr0,  TR::RealRegister::gr31, 0x7c00fed8u)
+));
```

Comment:
This should be using the `PPCTrg1MemEncodingTest` fixture instead of `PPCTrg1Src2EncodingTest`, as is done with the above instructions (the memory test will ensure that both `Trg1Mem` and `Trg1Src2` instructions work for index-form instructions).